grocery-list/src/App.js

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
 
 import './App.css';
+import ReusableComponent from './components/ResuableComponent1/ReusableComponent1'
```

Comment:
I think you can create an index file in the "Components" folder and use it for the exports and imports of all the components. Wouldn't it make it more readable and easy to use.
Like so: import { ComponentName } from './component'

Reply:
Thanks! That's a great suggestion. Will try to implement the same.